facade/service.go

```diff
@@ -2554,6 +2558,10 @@ func (f *Facade) GetServicesForScheduling(ctx datastore.Context, ids []string) [
 		if err != nil {
 			glog.Warningf("Could not get service with id %s: %s", id, err)
 		} else {
+			// CC-3871 preserve config changes after Emergency shutdown
+			if err := f.fillServiceConfigs(ctx, &svc); err != nil {
+				glog.Errorf("%s", err)
```

Comment:
Please replace the 2 glogs in this method with plog.WithError(err)....

Reply:
good